pkgs/development/python-modules/airlift/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, poetry-core, argparse, halo, pyyaml, hiyapyco, jinja2, dotmap, requests
+, kubernetes-helm, kind, docker, pythonOlder, buildPythonPackage
+, fetchFromGitHub }:
+
+buildPythonPackage rec {
+
+  disabled = pythonOlder "3.9";
+  pname = "airlift";
+  format = "pyproject";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "jl178";
+    repo = "airlift";
+    rev = "v${version}";
+    sha256 = "ZQ+YCMXQBM5Gb5O1jDQfKBeS1WxFKrtSRelgj65+f9U=";
```

```suggestion
    hash = "sha256-xxx";
```
> Nixpkgs contributors are currently recommended to use hash.
https://nixos.org/manual/nixpkgs/unstable/#fetchurl